ui/tests/integration/components/job-status-panel-test.js

```diff
@@ -0,0 +1,142 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { find, render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import { startMirage } from 'nomad-ui/initializers/ember-cli-mirage';
+import { initialize as fragmentSerializerInitializer } from 'nomad-ui/initializers/fragment-serializer';
+import { componentA11yAudit } from 'nomad-ui/tests/helpers/a11y-audit';
+
+module(
+  'Integration | Component | job status panel | active deployment',
+  function (hooks) {
+    setupRenderingTest(hooks);
+
+    hooks.beforeEach(function () {
+      fragmentSerializerInitializer(this.owner);
+      window.localStorage.clear();
```

Comment:
`nit`: You could probably move this into a `before` hook if you're concerned about other tests leaking into this one. You're already cleaning up after each hook.